code/__DEFINES/components.dm

```diff
@@ -104,6 +104,7 @@
 #define COMSIG_MOVABLE_MOVED "movable_moved"					//from base of atom/movable/Moved(): (/atom, dir)
 #define COMSIG_MOVABLE_CROSSED "movable_crossed"                //from base of atom/movable/Crossed(): (/atom/movable)
 #define COMSIG_MOVABLE_BUMP "movable_bump"						//from base of atom/movable/Bump(): (/atom)
+#define COMPONENT_BUMP_RESOLVED 1
```

Comment:
Put this tabbed under `#define COMSIG_MOVABLE_BUMP` (check other examples in this file) and use it in flag format (`(1<<0) instead of `1`).